src/pages/workspace/WorkspaceProfilePage.tsx

```diff
@@ -55,7 +59,7 @@ function WorkspaceProfilePage({policy, currencyList = {}, route}: WorkSpaceProfi
     const policyName = policy?.name ?? '';
     const policyDescription = policy?.description ?? '';
     const readOnly = !PolicyUtils.isPolicyAdmin(policy);
-    const imageStyle: StyleProp<ImageStyle> = isSmallScreenWidth ? [styles.mhv12, styles.mhn5] : [styles.mhv8, styles.mhn8];
+    const imageStyle: StyleProp<ImageStyle> = isSmallScreenWidth ? [styles.mhv12, styles.mhn5, styles.mbn5] : [styles.mhv8, styles.mhn8, styles.mbn5];
```

Comment:
Why does the image need a negative bottom margin? What if the top header image just didn't have bottom padding or something?

Reply:
Negative margin is need to place correctly all items under the image. Image is not having absolute position, but picker should be placed on it. Without margins items will be placed under each other - just like regular flex: 1, flex-direction: column